components/header/index.styl

```diff
@@ -92,11 +99,42 @@ header
   header
     height 75vh
 
+  .slide-1
+    &:before
+      content "some text 1"
+      position absolute
```

Comment:
Like we talked about, these duplicated styles should only need to be defined once. The only thing specific to these slide-n classes is the content.